packages/core/database/lib/index.js

```diff
@@ -118,7 +118,7 @@ class Database {
 // TODO: move into strapi
 Database.transformContentTypes = transformContentTypes;
 Database.init = async (config) => {
-  const db = new Database(config);
+  const db = await new Database(config);
```

Comment:
```suggestion
  const db = new Database(config);
  await db.init();
```
We don't want to be trying to use an async constructor. I recommend moving everything after createConnection to an async init() method and await it like this before returning.

Reply:
Updated.